src/types.ts

```diff
@@ -138,15 +138,16 @@ export interface IJupyterCadDoc extends YDocument<IJupyterCadDocChange> {
   addObjects(value: Array<IJCadObject>): void;
   updateObjectByName(name: string, key: string, value: any): void;
 
-  getOption(key: string): any;
-  setOption(key: string, value: any): void;
+  getOption(key: string): JSONValue;
```

Comment:
Since we fixed the `guiData` keyword, I think we can update the schema of `jcad` file to have a more explicit interface than the `JSONValue`

Reply:
I do not mean the content inside the `guiData`, but the `guiData` keyword now is a part of the `jcad` file specification, I think it's better to have it in the `jcad.json` schema file.